src/code42cli/cmds/search/__init__.py

```diff
@@ -21,10 +22,29 @@ def invoke(self, ctx):
         protocol = ctx.params.get("protocol")
         output_format = ctx.params.get("format", OutputFormat.RAW)
         ignore_cert_validation = ctx.params.get("ignore_cert_validation")
+        _handle_incompatible_args(protocol, ignore_cert_validation, certs)
+
         if ignore_cert_validation:
             certs = "ignore"
 
         ctx.obj.logger = _try_get_logger_for_server(
             hostname, protocol, output_format, certs
         )
         return super().invoke(ctx)
+
+
+def _handle_incompatible_args(protocol, ignore_cert_validation, certs):
+    if protocol == ServerProtocol.TLS_TCP:
+        return
+
+    arg = (
```

Comment:
Nested ternaries are a little hard to parse, could we just use if blocks?

Reply:
I did that originally but felt the opposite. I can put it back and show you what I mean

Reply:
Ah I guess it's not too bad, nevermind